src/BackupChain.cs

```diff
@@ -19,26 +20,28 @@ public class BackupChain : IBackupChain
   {
     private readonly IList<BackupMetadata> _orderedBackups;
 
+    // This also handles any striped backups
     private BackupChain(IList<BackupMetadata> recentBackups)
     {
+      if (!(recentBackups?.Count > 0)) {
+        throw new BackupChainException("There are no recent backups to form a chain");
+      }

```

Comment:
First off, if inputs are possibly null, then you should check that before all else.
Second - there is too many operators here: `if(recentBackups.empty())`

Reply:
`IList` doesn't have `Empty()`
https://github.com/factset/AgDatabaseMove/pull/62#discussion_r585817633

sure, will change to:
`if (recentBackups == null || recentBackups.Count == 0) { throw ... }`